src/metabase/api/common/internal.clj

```diff
@@ -55,16 +55,16 @@
 (defn- dox-for-schema
   "Look up the docstring for `schema` for use in auto-generated API documentation. In most cases this is defined by
   wrapping the schema with `with-api-error-message`."
-  [schema]
+  [schema route-str]
   (if-not schema
     ""
     (or (su/api-error-message schema)
         ;; Don't try to i18n this stuff! It's developer-facing only.
         (when config/is-dev?
           (log/warn
-           (u/format-color 'red (str "We don't have a nice error message for schema: %s\n"
+           (u/format-color 'red (str "We don't have a nice error message for schema: %s defined at %s\n"
```

Comment:
I added this because it was hard to track down which endpoint it was complaining about.

Reply:
Good idea!